Makefile

```diff
@@ -11,7 +11,7 @@ vtest: flake develop
 	py.test ./tests/
 
 cov cover coverage: flake
-	py.test --cov aiohttp_session --cov-report html ./tests/
+	py.test --cov aiohttp_session --cov-report html --cov-report=xml ./tests/
```

Comment:
It's not related to this PR but the preferred spelling of the `pytest` command is the one without the dot in the middle.